plugins/woocommerce/src/Internal/RestApi/Routes/V4/Settings/Account/Schema/AccountSettingsSchema.php

```diff
@@ -0,0 +1,303 @@
+			}
+		);
+
+		$response = array(
+			'id'          => 'account',
+			'title'       => __( 'Accounts & Privacy', 'woocommerce' ),
+			'description' => __( 'Set options relating to customer accounts and data privacy.', 'woocommerce' ),
+			'values'      => $values,
+			'groups'      => $groups,
+		);
+
+		if ( ! empty( $include_fields ) ) {
+			$response = array_intersect_key( $response, array_flip( $include_fields ) );
+		}
+
+		return $response;
+	}
+
+	/**
+	 * Transform a WooCommerce setting into REST API field format.
+	 *
+	 * @param array $setting WooCommerce setting array.
+	 * @return array|null Transformed field or null if should be skipped.
+	 */
+	private function transform_setting_to_field( array $setting ): ?array {
+		$setting_id   = $setting['id'] ?? '';
+		$setting_type = $setting['type'] ?? 'text';
+
+		$field = array(
+			'id'    => $setting_id,
+			'label' => $setting['title'] ?? $setting_id,
+			'type'  => $this->normalize_field_type( $setting_type ),
+			'desc'  => $setting['desc'] ?? '',
+		);
+
+		// Add options for select fields.
+		if ( isset( $setting['options'] ) && is_array( $setting['options'] ) ) {
+			$field['options'] = $setting['options'];
+		} else {
+			// Generate options for special field types.
+			$field['options'] = $this->get_field_options( $setting_id );
+		}
+
+		return $field;
+	}
+
+	/**
+	 * Get options for specific field types.
+	 *
+	 * @param string $field_id Field ID.
+	 * @return array Field options.
+	 */
+	private function get_field_options( string $field_id ): array {
+		// No field has options for now.
+		return array();
+	}
+
+	/**
+	 * Normalize WooCommerce field types to REST API field types.
+	 *
+	 * @param string $wc_type WooCommerce field type.
+	 * @return string Normalized field type.
+	 */
+	private function normalize_field_type( string $wc_type ): string {
+		$type_map = array(
+			'single_select_page'             => 'select',
+			'single_select_page_with_search' => 'select',
+		);
+
+		return $type_map[ $wc_type ] ?? $wc_type;
+	}
+
+	/**
+	 * Validate and sanitize field value based on its type.
+	 *
+	 * @param mixed  $value Field value.
+	 * @param string $type  Field type.
+	 * @return mixed Validated value.
+	 */
+	private function validate_field_value( $value, string $type ) {
+		switch ( $type ) {
+			case 'number':
+				return is_numeric( $value ) ? (float) $value : 0;
+			case 'checkbox':
+				if ( function_exists( 'wc_string_to_bool' ) ) {
+					return wc_string_to_bool( $value );
+				}
+				if ( is_bool( $value ) ) {
+					return $value;
+				}
+				return filter_var( $value, FILTER_VALIDATE_BOOLEAN );
+			case 'multiselect':
+				if ( ! is_array( $value ) ) {
+					return array();
+				}
+				return array_map( 'sanitize_text_field', $value );
+			case 'text':
+				return sanitize_text_field( $value );
+			case 'textarea':
+				return sanitize_textarea_field( $value );
+			case 'select':
+			default:
+				return sanitize_text_field( $value );
```

Comment:
Very minor suggestion, no functional difference just looks neater 😄  
```suggestion
			case 'textarea':
				return sanitize_textarea_field( $value );
			case 'text':	
			case 'select':
			default:
				return sanitize_text_field( $value );
```